tests/checks/integration/test_nginx.py

```diff
@@ -22,27 +22,42 @@ def setUp(self):
                 {
                     'nginx_status_url': 'http://localhost:44441/nginx_status/',
                     'tags': ['first_one'],
+                    'nginx_ssl_verify': True,
                 },
                 {
                     'nginx_status_url': 'http://dummyurl:44441/nginx_status/',
                     'tags': ['dummy'],
+                    'nginx_ssl_verify': True,
                 },
                 {
                     'nginx_status_url': 'http://localhost:44441/nginx_status/',
                     'tags': ['second'],
+                    'nginx_ssl_verify': True,
```

Comment:
Not that I know of. 

The nginx config we use in our CI test suite is located here : https://github.com/DataDog/dd-agent/blob/master/ci/resources/nginx/nginx.conf . 

If you do want to test that (it doesn't seem very useful indeed... but almost no test does until it breaks :) ) you'll have to create an HTTPS VirtualHost in Nginx in the conf file above and enable status on it. 

Oh and I don't think there's a need to enable `nginx_ssl_verify` for all the tests on http endpoints.